src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -74,6 +71,16 @@ const Body = BaseForm.extend(Object.assign(
     remove () {
       BaseForm.prototype.remove.apply(this, arguments);
       this.stopPolling();
+    },
+
+    triggerAfterError (model, error) {
+      BaseForm.prototype.triggerAfterError.apply(this, arguments);
+      this.stopPolling();
+
+      // Polling needs to be resumed if it's a form error and session is still valid
+      if(error.responseJSON.errorMessageKey !== 'idx.session.expired') {
```

Comment:
Curious if API can respond 409 (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429)
and we check error code at UI?

Reply:
I don't think making api changes are in the scope... Other simple thing we can do is just stop polling on any error and do not resume polling at all. In case of form error enduser will quickly correct the problem and would proceed, and in case if token has expired enduser will see a `session expired error`.. 

So what i am saying is we may not need to support this `resume poll on error` edge case.. What do you say?